src/core/include/cesium/omniverse/FabricMaterial.h

```diff
@@ -95,6 +103,15 @@ class FabricMaterial {
         uint64_t texcoordIndex,
         double alpha);
     void setImageryLayerAlphaValue(const omni::fabric::Path& path, double alpha);
+    void setFeatureIdIndexValues(const omni::fabric::Path& path, int nullFeatureId);
+    void
```

Comment:
This is very nit, but this seems like an odd way to divide up a line. Is this clang? The other function declaration and split at the parameter declarations.

Reply:
Yes it's clang-format :slightly_frowning_face: